libvirt.go

```diff
@@ -316,17 +316,28 @@ func (d *Driver) Start() error {
 	// They wont start immediately
 	time.Sleep(5 * time.Second)
 
-	for i := 0; i < 90; i++ {
-		time.Sleep(time.Second)
-		ip, _ := d.GetIP()
+	for i := 0; i <= 40; i++ {
```

Comment:
I would split all these changes from the getIPByMacFromSettings change as they are not related

Reply:
Yes I will do that.